.github/workflows/keepalive.yml

```diff
@@ -29,14 +29,22 @@ jobs:
           git merge --no-edit FETCH_HEAD &&
           echo "result=merged" >>$GITHUB_OUTPUT
       - name: Create a commit
+        id: commit
         if: steps.merge.outputs.result != 'merged'
         run: |
+          if test 0 -lt $(git rev-list --count --since=3.weeks.ago HEAD)
+          then
+            echo "No need to keep alive, there were commits in the last two weeks"
```

Comment:
_Some_ people may object to `last two weeks` and `--since=3.weeks.ago` for not being the same thing.